e2e/sample-apps/SQLiteSampleApp/DatabaseService.cs

```diff
@@ -113,7 +113,7 @@ public static int CreatePetByName(string petName)
                     }
                 }
             }
-            catch (SqliteException)
+            catch (SqliteException ex)
```

Comment:
rm if not used

Reply:
we want clean samplle apps, bc some clients just copy parts of the sample apps :))